connect-inject/container_init.go

```diff
@@ -103,14 +124,15 @@ func (h *Handler) containerInit(pod *corev1.Pod) (corev1.Container, error) {
 					FieldRef: &corev1.ObjectFieldSelector{FieldPath: "metadata.name"},
 				},
 			},
-		},
-		VolumeMounts: []corev1.VolumeMount{
-			corev1.VolumeMount{
-				Name:      volumeName,
-				MountPath: "/consul/connect-inject",
+			{
+				Name: "NAMESPACE",
```

Comment:
```suggestion
				Name: "POD_NAMESPACE",
```

Reply:
Good catch! I was wondering why I wasn't seeing this info...